src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -302,19 +302,19 @@ public override bool CanReadToken(string tokenString)
         /// Returns a Json Web Token (JWT).
         /// </summary>
         /// <param name="tokenDescriptor"> a <see cref="SecurityTokenDescriptor"/> that contains details of token contents and <see cref="SignatureProvider"/>.
-        public string CreateJwt(SecurityTokenDescriptor tokenDescriptor)
+        public string CreateEncodedJwt(SecurityTokenDescriptor tokenDescriptor)
```

Comment:
Yes, I want to make this now. Before we ship. I want the name of the method to distinguish if an encoded string OR JwtSecurityToken is returned.